etl/naaccr_etl_postgresql.sql

```diff
@@ -964,7 +970,9 @@ CREATE TABLE naaccr_data_points_temp
     , episode_source_concept_id
     , record_id
   )
-  SELECT ( CASE WHEN  (SELECT MAX(episode_id) FROM episode) IS NULL THEN 0 ELSE  (SELECT MAX(episode_id) FROM episode) END + row_number() over(order by cot.record_id))                 AS episode_id
+  SELECT COALESCE( (SELECT MAX(episode_id) FROM episode_temp)
```

Comment:
Again, this change does not cause any problems but also does not fix any problem.  Since this is the first insert into EPISODE_TEMP,  it is guaranteed to be empty.